Add 2022-12-07 agenda and minutes (#489)
solid/specification: meetings/2022-12-07.md

* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
* If this is your first time, welcome! please introduce yourself.

### Scribes
* Jeff Waters
* Wouter Termont

### Introductions
* name: text

---

## Topics

### Previous Meeting
URL: https://github.com/solid/specification/blob/main/meetings/2022-11-30.md

* SC: Consensus both proposing and how we accept them. We have prior consent, previous minutes are PR'd, once reviewed, we have approval but we link to acknowledge that to review or to raise objection to any part of prior meetings. Some concerns or comments, this is your chance. We acknowledge that. I'll pause if you have something to say. (none)

### Current Meeting
URL: https://github.com/solid/specification/pull/489

> * SC: Proposed agenda.

Reviewer: ```suggestion
```

* SC: We'll have time to talk about special topics, and we have topic of draft working group charter on Solid.

### W3C Calendar
URL: https://www.w3.org/groups/cg/solid/calendar

* SC: If you are not receiving notifications or updates, like when I update the event, you should be getting this from W3C, if not, let me know or review your own preferences, the steps are:
* SC: Email notifications of group events are enabled by default. Check settings:
  1. [Login](https://www.w3.org/accounts/login)
  2. [My Calendar](https://www.w3.org/users/myprofile/calendar)
  3. Preferences
  4. Check the box "Receive email notifications of events to which my groups are invited".
* SC: The announcements about the agenda are in the Gitter chat, as well as the PR. I'll pause if anyone has comments.
* PA: (muted)
* MB: I was wondering if it was possible to create one event that is recurring rather than so many notifications?
* SC: When the event is created and recurring, ... you will not get notification for recurring events unless the particular event is updated.
* MB: Ok. I'm wondering if that is what is happening, I receive too many of them.

> * SC: In order to include agenda in event calendar, I do need to update it.

Reviewer: I wasn't on the call, so this is not a change suggestion, but a relevant comment.

In most groups I'm in, we've found that including a *link* to the agenda in the calendar is better, because it goes in once, and any changes are just made to that remote document -- which do not trigger calendar notification email.  The agenda may be sent to the relevant email list, and its archive URL used for the link; or the agenda may continue to be pushed to GitHub, and a link to the relevant PR used in the calendar.

Author: Either way, a notification is distributed, whether by the W3C calendar system, an email to the mailing list, or some other way.

Alternatives:

* For people using the W3C CG's ICS, an update to the W3C calendar including the link to the agenda (without explicitly sending a notification out about the update) will suffice. They'll see the updated information when their local calendar synchronises with W3C's.

* A persistent URL for the agenda, e.g., `https://solidproject.org/w3c-solid-cg-upcoming-agenda` that redirects to the GitHub PR URL can be fixed in the W3C calendar. Requires a person or a script to change the redirect URL.

* SC: As you like. We can do that offline.
* eP: Interop panel moved to Monday, not Tuesday now.
* MB: We could potentially have authz, authn and interop alternating on Monday.
* SC: Any other comments on this? (none)

### Special Topic Meetings

* SC: Wanted to kick-off sporadic meetings with dedicated topics. We can have one on Use Cases next and perhaps a monthly/quarterly for an Introduction to the Solid CG.
* SC: These meetings can be at times different than the CG Weekly Call.
* SC: I want to introduce this special topic meetings, one is use cases generally speaking, follow-up from prior meetings and the next is introduction to Solid CG who is new and wants to learn. And we can have other special topics, on implementation level and so on. We'll see what's useful to everyone. Think about special topics you would like to propose.

#### Use Cases
URL: https://github.com/solid/specification/pull/488

* SC: Follows action https://github.com/solid/specification/blob/main/meetings/2022-11-23.md#use-cases.
* SC: Scheduled for 2022-12-14 (same time next week) but can be another datetime.

> * SC: The purpose of this is to focus on reviewing existing material that we have for use cases, to identify areas to improve to integrate use cases into what we do. This is not intended to discuss specific use cases, rather to discuss the process, to help with discovery and participants' concern about how we take use cases into account for spec development. This is a meeting about that and to improve that work, scheduled at this same time next week. So special topic on that.

Reviewer: ```suggestion
* SC: The purpose of this is to focus on reviewing existing material that we have for use cases, to identify areas to improve and integrate use cases into what we do. This is not intended to discuss specific use cases, rather to discuss the process, to help with discovery and participants' concern about how we take use cases into account for spec development. This is a meeting about that and to improve that work, scheduled at this same time next week. So special topic on that.
```

* ...: https://solidproject.solidcommunity.net/Verticals/Roadmap/index.ttl#this
* TBL: There is this history of the community thinking about this. A list of verticals. The way this works is that things.... People working about Solid code, we got people working on media, there is a project around fitness; there's a few things like issues trackers... there is an awful lot of things you expect to be in the project, a list of plans we have not yet committed to, a goal people have to meet for, government benefits, banking etc, and a lot of smaller things. Certainly there are things not on here. You can click on the pencil to put pointers to projects; let us know and we can all link it. Verticals are use cases for the Solid protocol. If you have use cases, they will be within these verticals.
* SC: Those are all very good inputs for next week. We should prepare to gather that input.  To pool some of the ideas; its good to have some institutional knowledge. Whether we're testing use cases or improving experience... I would like to update the agenda with the link you shared.

> * eP: I'll try to join the meeting. I think we can find some very common core scenario's, where we make some kind of assumptions, for example assuming users finding filesystem intuitive. In some of the panels we lean towards the user having the user have access policies managed in one place but also other approaches consider letting any app set access policies. All things considering user access policies, are good cases to look into user experience verticals.

Reviewer: ```suggestion
* eP: I'll try to join the meeting. I think we can find some very common core scenarios, where we make some kind of assumptions, for example assuming users finding filesystem intuitive. In some of the panels, we lean towards the user having access policies managed in one place, but also other approaches consider letting any app set access policies. All things considering user access policies are good cases to look into user experience verticals.
```

* SC: No, but HTML preview. You don't have a visualizer? /s
* SC: Something we can look at separately offline.
* MB (chat): You could use an HTML preview tool, for example [htmlpreview](https://htmlpreview.github.io/?https://github.com/barath/process/blob/draft-working-group-charter/draft-working-group-charter/index.html).
* WT: We should not tackle this but problem with all the specs, not just the PR versions, not available in a nice format. That's true, but the working group draft is really good, the text reads fluently, but as eP said, it's not concrete in certain points.
* SC: Thanks for sharing. Any other comments? Or additional announcements with the remaining 7 minutes?
* SC: Next week's special meeting on use cases and make change requests to the agenda. Or if preference for a different day, let me know.
* eP: I apologize for coming back, support for the draft which ones fit charter and which not, how would be the best step for doing that or should I come up with something to communicate which things would fit under charter for existing drafts?
* SC: The PR is the central place for that conversation, request changes or make comments or reach out to the author or director.

> * WT: I think there is at least one or two places in draft where key aspects are listed, he could add links to whatever specs we want to take with us there. That would be an idea.

Reviewer: ```suggestion
* WT: I think there is at least one or two places in draft where key aspects are listed, we could add links to whatever specs we want to take with us there. That would be an idea.
```

### Scribes
* name

### Introductions
* name: text

---

## Topics

### Previous Meeting
URL: https://github.com/solid/specification/blob/main/meetings/2022-11-30.md

### W3C Calendar
URL: https://www.w3.org/groups/cg/solid/calendar

* SC: Email notifications of group events are enabled by default. Check settings:
  1. [Login](https://www.w3.org/accounts/login)
  2. [My Calendar](https://www.w3.org/users/myprofile/calendar)
  3. Preferences
  4. Check the box "Receive email notifications of events to which my groups are invited".

### Special Topic Meetings

* SC: Wanted to kick-off sporadic meetings with dedicated topics. We can have one on Use Cases next and perhaps a monthly/quarterly for an Introduction to the Solid CG.
* SC: These meetings can be at times different than the CG Weekly Call.

#### Use Cases
URL: https://github.com/solid/specification/pull/488

* SC: Follows action https://github.com/solid/specification/blob/main/meetings/2022-11-23.md#use-cases .
* SC: This is a special topic meeting focusing on reviewing the existing material, and identifying areas to improve towards the development and integration of Use Cases.

> * SC: Scheduled for 2022-12-14 (time time next week) but can be another datetime.

Reviewer: ```suggestion
* SC: Scheduled for 2022-12-14 (same time next week) but can be another datetime.
```
